src/entity/src/metadata.js

```diff
@@ -34,12 +35,16 @@ export default class Metadata {
   }
 
   get emailVerification() {
-    return this.kmd.emailVerification.status;
+    if (isDefined(this.kmd.emailVerification)) {
```

Comment:
Yes beacuse if `emailVerification` is `null` or `undefined` it will throw and error.